### ISSUE TITLE
TrimMultilineRawString false positive: not a raw string...

### ISSUE THREAD

**Issue by TWiStErRob** [detekt/detekt#5428]
## Expected Behavior
No report

## Observed Behavior
```kotlin
val ctor = usableCtors.singleOrNull() ?: throw MappingException(
	"Unable to find unique constructor to instantiate $clazz using ${propertyValues.keys}\n${
		allCtors.joinToString("\n") { ctor -> ctor.toMismatchString(ctor in usableCtors, propertyValues) }
	}"
)
```
```
TrimMultilineRawString - [Multiline raw strings should be followed by `trimMargin()` or `trimIndent()`] at Heroku\backend\database\src\main\kotlin\net\twisterrob\neo4j\ogm\KotlinReflectionEntityInstantiator.kt:23:4
```

## Steps to Reproduce
`TrimMultilineRawStringSpec`:
```kotlin
    @Test
    fun `doesn't raise if it is not a raw string - multiline`() {
        val code = """
            val a = "Hello ${'$'}{
                "cruel"
            } world!"
        """.trimIndent()
        subject.compileAndLint(code)
        assertThat(subject.findings).isEmpty()
    }
```
## Context
There's no raw string on the horizon in the above code...

## Your Environment

* Version of detekt used: 1.22.0-RC2
* Link to your project (if it's a public repository): https://github.com/TWiStErRob/net.twisterrob.cinema/blob/9824524a20d128048f8ac8ea412828929f375b3f/Heroku/backend/database/src/main/kotlin/net/twisterrob/neo4j/ogm/KotlinReflectionEntityInstantiator.kt#L22-L26
